modules/apps/foundation/portal/portal-upload/src/main/java/com/liferay/portal/upload/configuration/settings/internal/UploadServletRequestSettingsImpl.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.upload.configuration.settings.internal;
+
+import com.liferay.portal.configuration.metatype.bnd.util.ConfigurableUtil;
+import com.liferay.portal.kernel.upload.UploadServletRequestSettings;
+import com.liferay.portal.upload.configuration.UploadServletRequestConfiguration;
+
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Modified;
+
+/**
+ * @author Pei-Jung Lan
+ */
+@Component(
+	configurationPid = "com.liferay.portal.upload.configuration.UploadServletRequestConfiguration",
+	immediate = true, service = UploadServletRequestSettings.class
+)
+public class UploadServletRequestSettingsImpl
+	implements UploadServletRequestSettings {
```

Comment:
The purpose of this class is to retrieve the configuration so that we can use it in core. We've always naming this type of classes and interfaces with Settings (i.e. `UserFileUploadsSettings`, `CaptchaSettings`, `OpenOfficeSettings`, etc...). However, you pointed out a good point. I agree this naming can create confusion with the "settings". I like your suggestion. I'll rename it and create a separate ticket to rename the other ones in portal.